docs/contributing/gatsby-style-guide.md

```diff
@@ -289,14 +291,16 @@ module.exports = {
 }
 ```
 
-##### `highlight-next-line`: highlights the next line
+##### **keyword**: `highlight-next-line`
+
+This keyword highlights the next line. Replace `keyword` with `highlight-next-line`.
 
 ````
 ```javascript:title=gatsby-config.js
 module.exports = {
 	siteMetadata: {
 		title: `GatsbyJS`,
-		// highlight-next-line
+		// keyword
```

Comment:
I suppose this _could_ get confusing. 

Any way we can keep the keyword and escape it so that the plugin doesn't actually highlight the snippet?

Reply:
@sidharthachatterjee my first thought was to include a screenshot but that seems like not a good pattern to start. I think the point of the keyword highlighting is to always highlight code blocks (pretty sure it's configured in `gatsby-remark-prismjs`).

Reply:
We could set the language to `none` to disable all syntax highlighting on that snippet, I guess. What do you think?

Reply:
oh good idea! let me try that